### PR TITLE
chore(docs): updated google tag manager link via themes build

### PR DESCRIPTION
**Pull request by venetrius**
related to: camunda/team-automation-platform/issues/160

### CODE REVIEW

File: themes/camunda/layouts/partials/footer.html [hunk 1]
```diff
@@ -8,7 +8,7 @@
           </p>
 
           <p>
-            <a href="http://camunda.org">camunda.org</a> and <a href="{{ .Site.BaseURL }}">docs.camunda.org</a> are part of Camunda 7
+            <a href="http://camunda.org">camunda.org</a> and <a href="{{ .Site.BaseURL }}">docs.camunda.org</a> are both part of Camunda
```
Comment (venetrius) [inline themes/camunda/layouts/partials/footer.html:11]: File was out of sync, changes in themes repo are later

Changes from docs-manual 2024.02.21: 
https://github.com/camunda/camunda-docs-manual/commit/e6f59b603f169a2fcadcb1b2491c7d0f9ec5bb1a

Changed in themes 2024.03.05:

https://github.com/camunda/camunda-docs-theme/commit/c0b25b74d30e425e3b9332c90ebc6ec92dceb2a3

File: themes/camunda/layouts/partials/header.html [hunk 2]
```diff
@@ -5,7 +5,7 @@
   <script>(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':
   new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0],
   j=d.createElement(s),dl=l!='dataLayer'?'&l='+l:'';j.async=true;j.src=
-  'https://www.googletagmanager.com/gtm.js?id='+i+dl;f.parentNode.insertBefore(j,f);
+  'https://ssgtm.camunda.org/gtm.js?id='+i+dl;f.parentNode.insertBefore(j,f);
```
Comment (venetrius) [inline themes/camunda/layouts/partials/header.html:8]: Intended change

File: themes/camunda/layouts/shortcodes/enterprise.html [hunk 3]
```diff
@@ -6,6 +6,6 @@ <h3>Enterprise Feature</h3>
   <p>
     Check the <a href="http://camunda.com/platform-7/editions/">Camunda enterprise homepage</a>
     for more information or get your
-    <a href="https://camunda.com/download/enterprise/">free trial version.</a>
+    <a href="http://camunda.com/download/enterprise/">free trial version.</a>
```
Comment (venetrius) [inline themes/camunda/layouts/shortcodes/enterprise.html:9]: This was updated 2 years ago in both repo: 
Themes was updated to use http: https://github.com/camunda/camunda-docs-theme/commit/ce6c4dd52144db3f398ae11c931ee3debf373b2d
Few days later manual docs updated with themes but manually changed to `https`
https://github.com/camunda/camunda-docs-manual/commit/fa996f4520fee937b07c63779f79a6f85488d4e1

### Todo
I will update the link in themes repo to `https`

Comment (venetrius) [reply themes/camunda/layouts/shortcodes/enterprise.html:9]: Updated docs theme to use `https` : https://github.com/camunda/camunda-docs-theme/pull/50

File: themes/camunda/layouts/partials/header.html [hunk 4]
```diff
@@ -34,13 +34,6 @@
   {{ $styles := resources.Get "css/docs.css" | fingerprint }}
 
   <link rel="stylesheet" href="{{ $styles.Permalink }}" integrity="{{ $styles.Data.Integrity }}">
-  {{- $permalink := .Permalink }}
```
Comment (venetrius) [inline themes/camunda/layouts/partials/header.html:37]: Synced from themes
See: https://github.com/camunda/camunda-docs-theme/pull/50#discussion_r1830988920

**Review by tasso94 — APPROVED** [review]
👍

**Pull request merged**